deploy/presto.yaml

```diff
@@ -0,0 +1,835 @@
+apiVersion: v1
```

Comment:
Does this collide with the OpenShift/e2e work in any way?

Reply:
I copied over the entire `deploy` directory from the POC repo.  I should remove the OCP template bits from this to make things clear what is being used in OpenShift and what is not.